GOVERNANCE.md

```diff
@@ -39,14 +39,15 @@ Changes to org maintainers use the following:
-* Org maintainers MUST remain active on the project. If they are unresponsive for > 3 months they will lose org maintainership unless a [super-majority](https://en.wikipedia.org/wiki/Supermajority#Two-thirds_vote) of the other org maintainers agrees to extend the period to be greater than 3 months
+* Org maintainers MUST remain active on the project. If they are unresponsive for > 3 months they will lose org maintainership unless a [two-thirds supermajority](https://en.wikipedia.org/wiki/Supermajority#Two-thirds_vote) of the other org maintainers agrees to extend the period to be greater than 3 months
 * Any eligible person may stand as an org maintainer by opening a [PR](https://github.com/notaryproject/.github/pulls).
 * When a PR is opened the project maintainers may vote
-  * The voting period will be open for a minimum of three business days and will remain open until a super-majority of project maintainers has voted
+  * The voting period will be open for a minimum of three business days and will remain open until a [two-thirds supermajority](https://en.wikipedia.org/wiki/Supermajority#Two-thirds_vote) of project maintainers has voted
   * Only current org maintainers are eligible to vote via casting a single vote each via a -1/+1 comment on the nomination issue or approving in GitHub.
-  * Once a [super-majority](https://en.wikipedia.org/wiki/Supermajority#Two-thirds_vote) has been reached the maintainer elect must complete [onboarding](#onboarding-a-new-maintainer) prior to becoming an official Notary Project maintainer.
+  * Once a [two-thirds supermajority](https://en.wikipedia.org/wiki/Supermajority#Two-thirds_vote) has been reached the maintainer elect must complete [onboarding](#onboarding-a-new-maintainer) prior to becoming an official Notary Project maintainer.
   * Once the maintainer onboarding has been completed a pull request is made on the repo adding the new maintainer to the [MAINTAINERS](MAINTAINERS) file.
-* When an org maintainer steps down, they become an emeritus maintainer
+* When an org maintainer steps down, they become an emeritus maintainer.
+* When the sole org maintainer representing a company decides to step down, ensuring a seamless transition and continued company involvement becomes crucial. In such cases, the company representative has the privilege to nominate a replacement from within their company. This nomination process involves [opening an issue on GitHub](https://github.com/notaryproject/.github/issues/new). Existing org maintainers are then eligible to cast their votes by expressing either support or dissent through a single comment using `+1` or `-1` on the nomination issue. To elect the new organizational maintainer, a [two-thirds supermajority](https://en.wikipedia.org/wiki/Supermajority#Two-thirds_vote) of votes in favor is required.
```

Comment:
What is `+1` and `-1`? Can we make it clearer?

Reply:
Updated